lib/templates/pullRequestOpened.hbs

```diff
@@ -1,36 +1,36 @@
+* ❌ Missing `CONTRIBUTORS.md` entry.
+  * Please add yourself to the [contributors]({{ contributorsUrl }}) file! 
 {{else}}
-@{{ userName }}, thanks for the pull request! Maintainers, we have a signed CLA from @{{ userName }}, so you can review this at any time.
-{{/if}}
+Thanks for the pull request @{{ userName }}! Let's see if this is ready for review.
 {{/if}}
+{{#if claEnabled}}
+{{#if errorCla}}
+* ⚠ There was an error checking the CLA! If this is your first contribution, please send in a [Contributor License Agreement](https://github.com/AnalyticalGraphicsInc/cesium/blob/master/CONTRIBUTING.md#contributor-license-agreement-cla).
+  * Maintainers, this was the error I ran into while attempting to process the CLA check. Please resolve it to continue CLA checking.
+  ```
+  {{ errorCla }}
+  ```
 {{else}}
-@{{ userName }}, thanks for the pull request!
+{{#if askForCla}}
+* ❌ Missing CLA. 
+  * Please send in a [Contributor License Agreement](https://github.com/AnalyticalGraphicsInc/cesium/blob/master/CONTRIBUTING.md#contributor-license-agreement-cla) (CLA) and comment back here to let us know to check this!
+{{else}}
+* ✔ Signed CLA found. 
+{{/if}}
 {{/if}}
-
-{{#if askAboutContributors}}
-I noticed you haven't added yourself to `CONTRIBUTORS.md`. Please add yourself to that file and comment back here to let us know it's ready! 
-
 {{/if}}
-
 {{#if askAboutChanges}}
-I noticed that [CHANGES.md]({{ repository_url }}/blob/master/CHANGES.md) has not been updated. If this change updates the public API in any way, fixes a bug, or makes any non-trivial update, please add a bullet point to `CHANGES.md` and comment on this pull request so we know it was updated. For more info, see the [Pull Request Guidelines]( https://github.com/AnalyticalGraphicsInc/cesium/blob/master/CONTRIBUTING.md#pull-request-guidelines).
-
+* ❔ [CHANGES.md]({{ repository_url }}/blob/master/CHANGES.md) was not updated.
+  * If this change updates the public API in any way, please add a bullet point to `CHANGES.md`.
 {{/if}}
 {{#if askAboutThirdParty}}
-I noticed that a file in one of our ThirdParty folders (`{{ thirdPartyFolders }}`) has been added or modified. Please verify that it has a section in [LICENSE.md]({{ repository_url }}/blob/master/LICENSE.md) and that its license information is up to date with this new version. Once you do, please confirm by commenting on this pull request.
-
+* ❔ Third party change was made. 
+  * Looks like a file in one of our ThirdParty folders (`{{ thirdPartyFolders }}`) has been added or modified. Please verify that it has a section in [LICENSE.md]({{ repository_url }}/blob/master/LICENSE.md) and that its license information is up to date with this new version. Once that's done, please comment back here to let us know!.
```

Comment:
This is the only bullet point that mentions commenting once updates have been made. I think we can just omit it for brevity's sake.